graylog2-web-interface/src/theme/docs/Fonts.md

```diff
@@ -2,19 +2,75 @@ Font settings are accesible from within `theme.fonts` object.
 
 ### Family - _`theme.fonts.family`_
 
-- **`body`**: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif
-- **`monospace`**: "Roboto Mono", Menlo, Monaco, Consolas, "Courier New", monospace
+```jsx noeditor
+import React, { useEffect } from 'react';
+import styled, { useTheme } from 'styled-components';
+
+import ColorSwatch, { Swatch } from './Colors';
+
+const { fonts } = useTheme();
+
+const FontFamilies = () => (
```

Comment:
Is this really a good example? What about using a simple styled component using a specific font?

Reply:
This is not intended to be used as an example for how to apply font families, it generates an overview of our available font families and sizes. It is not even possible to view this code in the documentation.

<img width="714" alt="image" src="https://user-images.githubusercontent.com/46300478/192753938-34307ef1-5363-4f55-80ae-80890be00276.png">

Before this change the lists were static and had to be update every time we make a change to the related theme settings.

Reply:
We could add an example on how to use them, but I think the examples we have in the `ThemeProvider & Usage` section is enough.

Reply:
Okay, got it! I think it would be helpful to render a text of that size for each entry, so you immediately see text sizes and their relations.

Reply:
Makes sense!

<img width="819" alt="image" src="https://user-images.githubusercontent.com/46300478/193221121-d83fa24e-5d47-4451-bfd0-25d1f6704c7b.png">

Reply:
Can we also include information about font-weights to use?

Reply:
We do not have font-weights in our theme and usually we just use `font-weight: bold;` or none. But if we want to add them and clean up the existing usage, we should do it in a follow-up PR.

Reply:
When I adjusted the font-sizes in sawmill I found something that was confusing to me:
I couldn't find a proper scaling for the `small` font-size. According to type scale there are 0.937 and 878rem when scaling with Minor Second, but no 0.889rem.
In sawmill the font-sizes are calculated based on the factor and the scale, I tried a lot but couldn't find a factor to get to 0.889rem - why did you decide on this font size?

Reply:
Good point I will adjust the small and navigation size in a follow-up PR.